RELEASES.md

```diff
@@ -1,3 +1,127 @@
+Version 1.30.0 (2018-10-25)
+==========================
+
+Language
+--------
+- [Procedural macros are now available.][52081] These kinds of macros allow for
+  more powerful code generation, there is a [new chapter available][proc-macros]
+  in Rust Programming Language book that goes further in depth.
+- [You can now use keywords as identifiers using the raw identifiers
+  syntax (`r#`).][53236] e.g. `let r#bool = true;`
+- [Using anonymous parameters in traits is now deprecated with a warning and
+  will be a hard error in the 2018 edition.][53272]
+- [You can now use `crate` in paths.][54404] This allows you to refer to the
+  crate root in the path. e.g. `use crate::foo;` refers to `foo` in `src/lib.rs`.
+- [`extern crate` definitions are now optional.][54404] To use a external crate
+  you can simply have `use serde;` and add the `serde` dependency in `Cargo.toml`
```

Comment:
`use serde;` is a part of 2018 edition, so it's not in 1.30 yet and `extern crate` items are still required for imports.
Stabilized `feature(extern_prelude)` works only for relative paths (`let x = my_crate::Thing();`), not imports.
Imports are still relative to the crate root on 2015 edition.

Reply:
https://github.com/rust-lang/rust/pull/49789 contains more detailed description and test examples.